app/src/main/java/io/gnosis/safe/notifications/NotificationManager.kt

```diff
@@ -40,21 +50,57 @@ class NotificationManager(
             }
         }
 
+    fun createNotificationChannelGroup(safe: Safe) {
+        val id = safe.address.asEthereumAddressChecksumString()
+        val name = safe.localName
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            notificationManager.createNotificationChannelGroup(NotificationChannelGroup(id, name))
+            createNotificationChannel(id)
```

Comment:
should notification channel id's have also some common prefix?

Reply:
It's not necessary. What would be an advantage?